modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/extension/dialog_definition.js

```diff
@@ -53,5 +53,94 @@ CKEDITOR.on('dialogDefinition', (event) => {
 		};
 
 		Liferay.once('destroyPortlet', clearEventHandler);
+
+		if (dialog.getName() === 'image2') {
+			var infoTab = dialogDefinition.getContents('info');
```

Comment:
I guess we can't use ecma6 in this extensions, right?

Reply:
right